client-mvp-04/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "vrms-client-mvp-04",
+  "version": "0.4.0",
+  "description": "VRMS Client MVP",
+  "author": "OlgaBilogurova",
+  "homepage": "https://www.vrms.io",
+  "license": "ISC",
+  "private": true,
+  "dependencies": {
+    "@babel/plugin-transform-react-jsx-self": "^7.10.4",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "classnames": "^2.2.6",
+    "cross-env": "^7.0.2",
+    "cross-var": "^1.1.0",
+    "d3": "^5.15.1",
+    "dotenv-cli": "^3.2.0",
+    "http-proxy-middleware": "^1.0.5",
+    "local-storage": "^2.0.0",
+    "mathjs": "^6.6.2",
+    "minimist": "^1.2.3",
+    "moment": "^2.24.0",
+    "moment-recur": "^1.0.7",
+    "node-sass": "^4.14.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.3",
+    "typescript": "^4.0.3"
```

Comment:
Are we writing this new frontend in Typescript?

Reply:
No, just JS. I tried to resolve all warnings and set it up. Thank you for catching it. It's better to uninstall unuseful dependency. 👍